templates/nextflow-s3-template.yaml

```diff
@@ -0,0 +1,177 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Private S3 Bucket for Nextflow
+
+# Parameters
+Parameters:
+  Department:
+    Description: 'The department for this resource'
+    Type: String
+    AllowedPattern: '^\S*$'
+    ConstraintDescription: 'Must be string with no spaces'
+  Project:
+    Description: 'The name of the project that this resource is used for'
+    Type: String
+    AllowedPattern: '^\S*$'
+    ConstraintDescription: 'Must be string with no spaces'
+  OwnerEmail:
+    Description: 'Email address of the owner of this resource'
+    Type: String
+    AllowedPattern: '^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$'
+    ConstraintDescription: 'Must be an acceptable email address syntax(i.e. joe.smith@sagebase.org)'
+  BucketName:
+    Type: String
+    Description: (Optional) Name of the created bucket.
+  ReadWriteAccessArns:
+    Type: CommaDelimitedList
+    Default: "[]"
```

Comment:
You've confirmed that this should be quoted, right?

Reply:
@BrunoGrandePhD Yes! If it's not quoted, it returns this error:
`"An error occurred (ValidationError) when calling the UpdateStack operation: Template format error: Every Default member must be a string."`

Reply:
It seems like `[]` is just a flag value for indicating that nothing was provided and is not being interpreted as an empty list. 

I would try dropping the two `Default: "[]"` lines from this template and replace `"[]"` in the conditions with the empty string `""`. If that doesn't work, then I would add back `Default: ""`.

Reply:
Dropping the Default makes the parameter required. Added `Default: ""`